ansible/plugins/inventory/yacloud_compute.py

```diff
@@ -0,0 +1,154 @@
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# taken and changed from https://github.com/rodion-goritskov/yacloud_compute
+
+from __future__ import (absolute_import, division, print_function)
+from pathlib import Path
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
```

Comment:
as an alternative ![](https://steamcommunity-a.akamaihd.net/economy/emoticon/:ohh_yeah:)
```py
"""
makes the funny.
"""

from amogus.imposter import sus

print(__doc__)
```
will print `makes the funny.`

Reply:
Btw you could install the plugin to `$HOME/.ansible/plugins/inventory` to avoid checking it into the Git